packages/core/src/features/extensions/enabled/main/storage-version.injectable.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import storageMigrationVersionInjectable from "../../../../common/persistent-storage/storage-migration-version.injectable";
+import { enabledExtensionsMigrationDeclarationInjectionToken } from "../common/migrations";
```

Comment:
Do these need to be in `common` still? Could be `main`?

Reply:
What do you mean by "in common", this file is under `src/features/**/main/` which means it only gets included in `main`

Reply:
I'm referring to `storageMigrationVersionInjectable` and `enabledExtensionsMigrationDeclarationInjectionToken`. Do they still need to be in `common`? Is there a need or them in renderer?

Reply:
Ah I see and yes I think those should probably be moved